src/main.py

```diff
@@ -63,20 +63,7 @@ def check_fee_ini(args=None):
     return 1
 
 
-def check_ithaca_live(args=None):
```

Comment:
we should rename this function. The context: After a protocol upgrade people should not use TRD without carefully testing. normally we have a specific test branch with an open PR and ask people to go there. If they still use the old master at least this "end of service statement" is telling them to be careful. It is not elegant but we could update the date here for the date we assume Lima will go live.